.github/workflows/ci_push_testing.yml

```diff
@@ -0,0 +1,223 @@
+        chmod u+x dirac-install
+        DIRAC_VERSION=`curl -s https://raw.githubusercontent.com/DIRACGrid/DIRAC/integration/release.notes | grep -oP "\\[\\Kv[6]r.*[^\\]]" | grep -m1 -v -e "pre"`
+        ./dirac-install -r $DIRAC_VERSION -i 27 -g v14r1
+        source ~/dirac_ui/bashrc
+        dirac-proxy-init -x
+        dirac-configure -F -S GridPP -C dips://dirac01.grid.hep.ph.ic.ac.uk:9135/Configuration/Server -I
+        dirac-proxy-init -g gridpp_user -M
+    - name: Test with pytest
+      run: python3 -m pytest --cov-report term-missing --cov ganga/GangaDirac ganga/GangaDirac/test/Unit
+
+  dirac-integration:
+    name: GangaDirac Integration
+    needs: dirac-unit
+    runs-on: ubuntu-latest
+    container: centos:7
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3
+      run: yum install -y python3
+    - name: Install dependencies
+      run: |
+        python3 -m pip install --upgrade pip setuptools wheel
+        python3 -m pip install -e .
+        python3 -m pip install coverage pytest pytest-cov pytest-pylint pytest-mock
+    - name: Install Robot certificate
+      env: # Or as an environment variable
+        ROBOT_CERT: ${{ secrets.GangaRobot_UserCert }}
+        ROBOT_KEY: ${{ secrets.GangaRobot_UserKey }}
+      run: |
+        mkdir ~/.globus
+        echo "$ROBOT_CERT" > ~/.globus/usercert.pem
+        echo "$ROBOT_KEY" > ~/.globus/userkey.pem
+        chmod 644 ~/.globus/usercert.pem
+        chmod 400 ~/.globus/userkey.pem
+    - name: Install DIRAC UI
+      run: |
+        yum install -y wget
+        mkdir ~/dirac_ui
+        cd ~/dirac_ui
+        wget -np -O dirac-install https://raw.githubusercontent.com/DIRACGrid/DIRAC/integration/Core/scripts/dirac-install.py
+        chmod u+x dirac-install
+        DIRAC_VERSION=`curl -s https://raw.githubusercontent.com/DIRACGrid/DIRAC/integration/release.notes | grep -oP "\\[\\Kv[6]r.*[^\\]]" | grep -m1 -v -e "pre"`
+        ./dirac-install -r $DIRAC_VERSION -i 27 -g v14r1
+        source ~/dirac_ui/bashrc
+        dirac-proxy-init -x
+        dirac-configure -F -S GridPP -C dips://dirac01.grid.hep.ph.ic.ac.uk:9135/Configuration/Server -I
+        dirac-proxy-init -g gridpp_user -M
+    - name: Install gangarc file
+      run: |
+        echo -e "[DIRAC]\nDiracEnvSource = ~/dirac_ui/bashrc" > ~/.gangarc
+        echo -e "[Configuration]\nRUNTIME_PATH=GangaDirac" >> ~/.gangarc
+        echo -e "[defaults_DiracProxy]\ngroup=gridpp_user" >> ~/.gangarc
+    - name: Test with pytest
+      env:
+        GANGA_CONFIG_FILE: ~/.gangarc
+        GANGA_CONFIG_PATH: GangaDirac/Dirac.ini
+        DIRAC_DEPRECATED_FAIL: True
+      run: python3 -m pytest --cov-report term-missing --cov ganga/GangaDirac ganga/GangaDirac/test/GPI
+
+  lhcb-unit:
+    name: GangaLHCb Unit
+    needs: lint
+    runs-on: ubuntu-latest
+    container: 
+      image: centos:7
+      options: --privileged
```

Comment:
Is this still required - or was it only added during the testing and then not removed again when different approach was taken?

Reply:
Yes it is still required, without it the fuse mount of cvmfs doesn't work. I note also we use it in the current Jenkins tests too.i assume for a similar reason